examples/user_interaction/client/src/App.js

```diff
@@ -27,6 +28,10 @@ class App extends React.Component {
   }
 
   handleClick() {
+    // Start a child span for the Promise, this span will be son of the 
+    // current root span related to the current user interaction.
+    // These spans should be created in the code the click handler will run.
+    const childSpan = tracing.tracer.startChildSpan({ name: 'promise' });
```

Comment:
Could we give a more descriptive name to this custom span? What is the operation doing that it is measuring? I would think the more descriptive name could be given to both the `childSpan` variable and the `name:` field